WordPressAuthenticator/Signin/LoginPrologueLoginMethodViewController.swift

```diff
@@ -0,0 +1,73 @@
+import WordPressUI
+import WordPressShared
+
+class LoginPrologueLoginMethodViewController: NUXViewController {
+    /// Buttons at bottom of screen
+    private var buttonViewController: NUXButtonViewController?
+
+    /// Gesture recognizer for taps on the dialog if no buttons are present
+    fileprivate var dismissGestureRecognizer: UITapGestureRecognizer?
+
+    open var emailTapped: (() -> Void)?
+    open var googleTapped: (() -> Void)?
+    open var selfHostedTapped: (() -> Void)?
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
+
+        if let vc = segue.destination as? NUXButtonViewController {
+            buttonViewController = vc
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureButtonVC()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+
+    private func configureButtonVC() {
+        guard let buttonViewController = buttonViewController else {
+            return
+        }
+
+        let wordpressTitle = NSLocalizedString("Continue with WordPress.com", comment: "Button title. Tapping begins our normal log in process.")
+        let googleTitle = NSLocalizedString("Continue with Google", comment: "Button title. Tapping begins log in using Google.")
+        
+        buttonViewController.setupTopButton(title: wordpressTitle, isPrimary: false, accessibilityIdentifier: "Log in with Email Button") { [weak self] in
+            self?.dismiss(animated: true)
```

Comment:
Should we be tracking a tap on this button too?

Reply:
Sorry, I didn't think to make a note. I'm keeping a list of new tracks that are needed. I"ll add them a bit later.